jumanji/environments/routing/connector/env.py

```diff
@@ -70,17 +70,16 @@ class Connector(Environment[State]):
         - each value in the array corresponds to an agent's action.
 
     - reward: jax array (float) of shape ():
-        - dense: reward is increased by 1 for each successful connection on that step. Additionally,
+        - dense: reward is 1 for each successful connection on that step. Additionally,
             each pair of points that have not connected receives a penalty reward of -0.03.
 
-    - episode termination: if an agent can't move, or the time limit is reached, or the agent
-        connects to its target, it is considered done. Once all agents are done, the episode
-        terminates. The timestep discounts are of shape (1,) and only set to `0` when all agents
-        are done.
+    - episode termination:
+        * all agents either can't move (no available actions) or have connected to their target.
+        * the time limit is reached.
```

Comment:
```suggestion
        - all agents either can't move (no available actions) or have connected to their target.
        - the time limit is reached.
```
Do we use `*` or `-`?